crates/provider-wadm/wit/interfaces.wit

```diff
@@ -0,0 +1,6 @@
+package wasmcloud:provider-wadm;
+
+world interfaces {
+    import wasmcloud:messaging/handler@0.2.0;
```

Comment:
Not sure if I follow the logic for importing handler here. Is the goal that a status message could trigger a component? If that is the case, would it be better to have an interface with a concrete type that this provider can call and a component can implement?

Reply:
Added status update wit-specific type